packages/kbn-test/tsconfig.json

```diff
@@ -1,24 +1,17 @@
 {
   "extends": "../../tsconfig.base.json",
   "compilerOptions": {
-    "incremental": false,
+    "allowJs": true,
```

Comment:
@jbudz I think this is the cause of the issues, several JS files are used here to avoid types from mocha and jest from both being loaded.